### DISCUSSION TITLE
Restart HDF5 error

### DISCUSSION THREAD

**Discussion by ToxFox88** [damask-multiphysics/DAMASK#30] [Q&A] [answered]
Hello! I am trying to study regridding via restart option and have some troubles with it. 
There are two setups I tried: 
1. The two_phase example from https://github.com/MarDiehl/regridding which both falls with an error on 'DAMASK_grid -l tensionX.yaml -g simple.vti -r 14' command and regridding problem with py script (I suppose the problem is this script is 2 y.o. and I try to fix it) which is connected with (I suppose) the wrong order of 'sovler/F' array.
2. My own two phase material made recently on the bigger geometry so I'm sure it is not obsolete. 
 Both of them falls with an error when damask tries to rewrite the existing increment. e.g. I did 19 increments then return to 10 by restart option, it counts increment 11 and falls with an error. If I set f_out=2, it count 11, 12 and falls when it tries to save results. 
The error I get is attiched.
Are there any problems with restart option or I did something wrong? I tried on beta2 and stable version. 
And is there a working version (any scripts?) of the regridding algorithm?

[error.txt](https://github.com/user-attachments/files/17116382/error.txt)

**Comment by hyharry** [comment]
when using `f_restart` the simulation will append result in existing `result.hdf5` file. If a recording already exists there, hdf5 backend will then complain with error. One example is `f_out: 2` and `f_restart: 10`, suppose simulation stops at increment 13, if restart from increment 10, the resumed simulation will try to overwrite the existing increment 12 in your `result.hdf5`, then DAMASK will error out. 

One can delete every recorded load increment in `result.hdf5` after `n_restart` before restarting, then hdf5 will not complain.

**Reply by MarDiehl** [reply]
please forget about the regridding repo, it's outdated. I forgot about it and will delete it now.

**Reply by MarDiehl** [reply, marked as answer]
https://github.com/MarDiehl/regrid works.

**Reply by ToxFox88** [reply]
This link is 404. Looks like it is not public and I can't get an access to it.

**Reply by MarDiehl** [reply]
sorry, it was private. Should be visible now.

**Reply by ToxFox88** [reply]
Thanks a lot!
